app/models/owner.rb

```diff
@@ -74,8 +74,7 @@ def stripe_price_id
     main_price_id = ENV['STRIPE_SUBSCRIPTION_PRICE_ID']
 
     raise "ENV['STRIPE_SUBSCRIPTION_PRICE_ID'] is empty" unless main_price_id.present?
-
-    Affiliate.find_by(code: affiliate)&.stripe_price_id_monthly || main_price_id
+    affiliate.stripe_price_id_monthly || main_price_id
```

Comment:
```suggestion
    affiliate&.stripe_price_id_monthly || main_price_id
```
Elsewise this will also crash for empty affiliates